### ISSUE TITLE
Komanda on Linux?

### ISSUE THREAD

**Issue by curiositry** [mephux/komanda#208]
Do you plan to support linux — make a .deb available / add it to the ubuntu software centre / make a ppa?

Thanks!

**Comment by bendem** [comment]
Yep, that fixes it on fedora as well: `sudo ln -s /usr/lib64/libudev.so.1 /usr/lib64/libudev.so.0`

`yum provides libudev.so.0` doesn't seem to find packages providing it so I guess that's a way to fix it.
